test/binder/binder_test.cpp

```diff
@@ -167,7 +181,10 @@ TEST_F(BinderCorrectnessTest, SelectStatementTest) {
 
   // Test alias and select_list
   LOG_INFO("Checking select_list and table alias binding");
-  binder.reset(new binder::BindNodeVisitor());
+  delete binder->txn;
```

Comment:
Why raw delete here? If this is cleanup, it shouldn't happen here. More importantly, I don't think this is correct.